src/shop-app.html

```diff
@@ -441,7 +441,7 @@
             Polymer.importHref(this.resolveUrl('lazy-resources.html'), () => {
               // Register service worker if supported.
               if ('serviceWorker' in navigator) {
-                navigator.serviceWorker.register('/service-worker.js');
+                navigator.serviceWorker.register('service-worker.js', {scope: '/'});
```

Comment:
How does setting `base` in a document affect resource loading in a service worker? Suppose I have customized the service worker so that I can A/B test a specific component by redirecting requests for it to one of two candidate implementations. Will I need to be aware of the base configuration in order to correctly load the candidate implementation that has been chosen?